spec/spec_helper.rb

```diff
@@ -85,7 +62,6 @@ def image_path(path, options = {})
 
   config.before(:each) do |example|
     EmsRefresh.try(:debug_failures=, true)
-    ApplicationController.handle_exceptions = false if %w(controller requests).include?(example.metadata[:type])
```

Comment:
..until a spec fails. Then you'll see a generic 500 instead of the actual exception..